people_options.rb

```diff
@@ -0,0 +1,49 @@
+require './student'
+require './teacher'
+
+class PeopleOptions
+  attr_accessor :people
+
+  def initialize
+    @people = []
+  end
+
+  def list_all_people
+    puts "There is not a person yet...\n" if @people.count.zero?
+
+    @people.each do |person|
+      puts "[#{person.class}] ID: #{person.id} Name: #{person.name} Age: #{person.age}"
+    end
+  end
+
+  # rubocop:disable Metrics/MethodLength
```

Comment:
- [ ] You are not allowed to disable linter, Linter is a tool for identifying and reporting errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows, with the goal of making code more consistent and avoiding bugs. Kindly fix this instead of disabling it 👍🏻 